e2e/test.config.js

```diff
@@ -22,7 +22,7 @@ const PROXY = process.env.PROXY_HOST_ADF;
 const LOG = process.env.LOG;
 const BPM_HOST = process.env.URL_HOST_BPM_ADF || "bpm";
 const OAUTH_HOST = process.env.URL_HOST_SSO_ADF || "keycloak";
-const OAUTH_CLIENT_ID = process.env.OAUTH_CLIENDID || "alfresco";
+const OAUTH_CLIENT_ID = process.env.OAUTH_CLIENDID || "activiti";
```

Comment:
why you are changing the default client it?

Reply:
Isn't it supposed to be activiti, instead of alfresco ?

Reply:
There is a misspell in the environment variable name `OAUTH_CLIENTID` instead `OAUTH_CLIENDID`